react/index.js

```diff
@@ -83,3 +83,8 @@ export { default as Popup } from './Popup'
 export { default as IconStack } from './IconStack'
 export { default as QuotaAlert } from './QuotaAlert'
 export { default as ThresholdBar } from './ThresholdBar'
+export {
+  default as ViewStack,
+  useViewStack,
+  ViewStackContext
```

Comment:
We export both `useViewStack` and `ViewStackContext` to give the possibility to the user to use the hook or the context consumer? If that's it, shouldn't we expose only the hook so there is only one API?

Reply:
The hook is only usable in functional components, this is why I exported `ViewStackContext`.